src/components/select/utils/is-expected-option.js

```diff
@@ -0,0 +1,17 @@
+import Option from "../option/option.component";
+
+export default function isExpectedOption(element, expectedValue) {
```

Comment:
If you pass the `value` here instead of an `element` you could use this function instead of writing `isExpectedValue`

Reply:
that would not work as I have to check if it's an Option first.
I will extract part of the code into isExpectedValue method that could be reused elsewhere.

Reply:
Yeah probably more readable than filtering the array anyway